asv_bench/benchmarks/benchmarks.py

```diff
@@ -43,12 +43,7 @@ class BaseTimeGroupBy:
     def setup(self, shape, ngroups=5, groupby_ncols=1):
         ngroups = translator_groupby_ngroups(ngroups, shape)
         self.df, self.groupby_columns = generate_dataframe(
-            "int",
-            *shape,
-            RAND_LOW,
-            RAND_HIGH,
-            groupby_ncols,
-            count_groups=ngroups,
+            "int", *shape, RAND_LOW, RAND_HIGH, groupby_ncols, count_groups=ngroups,
```

Comment:
@sofezticated-gentleman I haven't taken a look at why you're running into the issues with `mypy` yet, but your PR seems to be touching a bunch of unrelated files. I assume this is because you are using a different linter. Could you make sure that you are following the instructions here: https://modin.readthedocs.io/en/stable/development/contributing.html#code-formatting-and-lint

Reply:
+1 @sofezticated-gentleman thank you for making this PR. I'm happy to help you merge it, but could you start by reverting all the formatting changes like this? Did you make these edits manually or did your editor make them automatically? If you have an IDE like vscode, I strongly recommend that you use black to autoformat your code, as in the steps here: https://dev.to/adamlombard/how-to-use-the-black-python-code-formatter-in-vscode-3lo0

Reply:
yeah I have been using the black formatter there were some conflicts, so I did these this manually but now I have corrected it